300-quarkus-vertx-webClient/README.md

```diff
@@ -0,0 +1,40 @@
+# Quarkus Vertx Webclient Ext
+
+## Recommended Readings
+
+[Vert.x Web Clients](https://quarkus.io/guides/vertx#using-vert-x-clients)
+
+## Requirements
+
+To compile and run this demo you will need:
+
+- JDK 1.8+
+
+## Scope of the testing
+
+Vert.x Mutiny webClient exploratory test. 
+* Vert.x WebClient
+* Quarkus Resteasy Mutiny / Jsonb
+* Quarkus configuration converters
+* Exception mapper
+* Automating testing
+
+## Building the application
+
+Launch the Maven build on the checked out sources of this demo:
+
+> ./mvnw install
```

Comment:
To do not rely on the .mvn folder, I think it's better have maven as a requirement and just use the mvn command.

Reply:
Agree, I think info about install / quarkus:dev can be scratched from the readme, it's quite natural thin for any Quarkus app. Maybe just a note in root README with pointer to https://quarkus.io/guides/getting-started-mode or https://quarkus.io/guides/getting-started#development-mode